gravitee-apim-portal-webui-next/src/app/api-details/api-tab-details/api-tab-details.component.ts

```diff
@@ -13,13 +13,52 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import { Component } from '@angular/core';
+import { AsyncPipe, DatePipe } from '@angular/common';
+import { Component, Input, OnInit } from '@angular/core';
+import { map, Observable, of, switchMap } from 'rxjs';
+
+import { MarkdownDescriptionPipe } from '../../../components/pipe/markdown-description.pipe';
+import { MarkdownService } from '../../../components/pipe/markdown.service';
+import { ConfigService } from '../../../services/config.service';
+import { PageService } from '../../../services/page.service';
 
 @Component({
   selector: 'app-api-tab-details',
   standalone: true,
-  imports: [],
+  imports: [MarkdownDescriptionPipe, AsyncPipe, DatePipe],
   templateUrl: './api-tab-details.component.html',
   styleUrl: './api-tab-details.component.scss',
 })
-export class ApiTabDetailsComponent {}
+export class ApiTabDetailsComponent implements OnInit {
+  @Input() apiId!: string;
+  @Input() apiVersion!: string;
+  @Input() apiLastUpdate!: Date | undefined;
+  @Input() apiCategories: Array<string> | undefined;
+  content$!: Observable<string | null | undefined>;
+
+  constructor(
+    private pageService: PageService,
+    private configurationService: ConfigService,
+    private markdownService: MarkdownService,
+  ) {}
+
+  ngOnInit(): void {
+    this.content$ = this.pageService.listByApiId(this.apiId, true).pipe(
+      switchMap(pageResponse => {
+        if (pageResponse.data) {
+          return this.pageService.content(this.apiId, pageResponse.data[0].id).pipe(
+            map(response => {
+              if (response.content) {
+                return this.markdownService.render(response.content, this.configurationService.baseURL, '/documentation/root', []);
+              } else {
+                return null;
+              }
+            }),
+          );
+        } else {
+          return of(null);
+        }
+      }),
+    );
```

Comment:
Best to avoid using a pipe within a pipe. What do you think of this?
```suggestion
  this.content$ = this.pageService.listByApiId(this.apiId, true).pipe(
      switchMap(pageResponse => {
        if (pageResponse.data) {
          return this.pageService.content(this.apiId, pageResponse.data[0].id);
        } else {
          return of(null);
        }
      }),
      map(pageWithContent => {
        if (pageWithContent?.content) {
          return this.markdownService.render(pageWithContent.content, this.configurationService.baseURL, '/documentation/root', []);
        } else {
          return null;
        }
      }),
    );
```

Reply:
Done